### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by sunilshahi**
function name 'fn_cdc_map_time_lsn' should be 'fn_cdc_map_time_to_lsn'

### CODE REVIEW

**Comment by PRMerger10** [comment]
@sunilshahi : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/relational-databases/system-functions/sys-fn-cdc-map-time-to-lsn-transact-sql.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ sys.fn_cdc_map_time_to_lsn ( '<relational_operator>', tracking_time )
...
+ To understand how the **sys.fn_cdc_map_time_to_lsn** can be used to map datetime ranges to LSN ranges, consider the following scenario. Assume that a consumer wants to extract change data on a daily basis. That is, the consumer wants only changes for a given day up to and including midnight. The lower bound of the time range would be up to but not including midnight of the previous day. The upper bound would be up to and including midnight of the given day. The following example shows how the function **sys.fn_cdc_map_time_to_lsn** can be used to systematically map this time-based range into the LSN-based range needed by the change data capture enumeration functions to return all changes within that range.
```
Comment (rwestMSFT) [inline docs/relational-databases/system-functions/sys-fn-cdc-map-time-to-lsn-transact-sql.md:59]:
```suggestion
 To understand how the `sys.fn_cdc_map_time_to_lsn` function can be used to map datetime ranges to LSN ranges, consider the following scenario. Assume that a consumer wants to extract change data on a daily basis. That is, the consumer wants only changes for a given day up to and including midnight. The lower bound of the time range would be up to but not including midnight of the previous day. The upper bound would be up to and including midnight of the given day. The following example shows how the function `sys.fn_cdc_map_time_to_lsn` can be used to systematically map this time-based range into the LSN-based range needed by the change data capture enumeration functions to return all changes within that range.  
```

**Comment by jborsecnik** [comment]
@rwestMSFT, 

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by rwestMSFT** [comment]
@jborsecnik I made a suggested change. Once that's implemented I'll approve the PR.

**Comment by jborsecnik** [comment]
> @jborsecnik I made a suggested change. Once that's implemented I'll approve the PR.

PR team can also make those changes if you want. If so, let us know and sign off. Thx!

**Comment by rwestMSFT** [comment]
@jborsecnik If you can make that minor tweak then please do!

**Pull request closed without merging**